File: PHP/22-02-2021/calculation.php
<?php
class Calculation {
    private $x;
    private $y;
    public function Calculation($x, $y) {
        $this->x = $x;
        $this->y = $y;
    }
    //instance method = 
    public function sum() {
        return $this->x + $this->y; 
    }
    public function substract() {
        return $this->x - $this->y; 
    }
    public function multiply() {
        return $this->x * $this->y; 
    }
    public function divide () {
        return $this->x + $this->y; 
    }
}
?>

File: ADF1/21-05-2021/src/main/java/Calculation.java
public class Calculation {
    //thuc hien cong, tru, nhan, chia
    //Moi class la 1 file .java
    public Integer sum(Integer x, Integer y) {
        return x + y;
    }
    public Integer minus(Integer x, Integer y) {
        return x - y ;
    }
    public Integer multiply(Integer x, Integer y) {
        return x * y;
    }
    public Integer divide(Integer x, Integer y) {
        return y == 0 ? 0  : x / y; //neu nhap y = 0, chuong trinh se crash
        //return  x / y;
    }
    public void doSomething() {
        System.out.println("do something");
        //for loop giong C
        for(int i = 0; i < 100_000_000; i++) {
            System.out.print(String.format("i = %d, ", i));
            if(i % 9 == 0) {
                System.out.println();
            }
        }
    }
}
